server.go

```diff
@@ -172,17 +153,17 @@ func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
 		LocalChannelClose: func(pubKey []byte,
 			request *htlcswitch.ChanClose) {
 
-			s.peersMtx.RLock()
-			peer, ok := s.peersByPub[string(pubKey)]
-			s.peersMtx.RUnlock()
-
-			if !ok {
+			peer, err := s.FindPeerByID(string(pubKey))
+			if err != nil {
 				srvrLog.Error("unable to close channel, peer"+
-					" with %v id can't be found", pubKey)
+					" with %v id can't be found: %v", pubKey, err)
```

Comment:
With 4-space tabs I only see this as 66 characters, are we doing our counts using 8-space tabs?